warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java

```diff
@@ -33,9 +33,11 @@
 import datawave.query.jexl.functions.arguments.JexlArgumentDescriptor;
 import datawave.query.jexl.nodes.QueryPropertyMarker;
 import datawave.query.jexl.visitors.EventDataQueryExpressionVisitor;
+import datawave.query.jexl.visitors.PrintingVisitor;
```

Comment:
is this used? I dont see it in the changes to this file, but I may have just missed it.

Reply:
It is not being used. Just pushed a commit to remove it.